independent-publisher-connectors/Workable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1692 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Workable",
+    "description": "The Workable API can be used in the following ways:\n\nExtract information about your:\naccount\nhiring members\npipeline stages\njobs \nscheduled events\njob's and/or candidate's activity\n\nManage your candidates by:\nFetching candidates\nExtracting detail candidate info",
```

Comment:
@DavidBreathe please remove the "API" word and the newline "\n" from the description. The description should meet our [requirements](https://learn.microsoft.com/en-gb/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

Reply:
API word removed